R/teal.R

```diff
@@ -14,7 +14,7 @@
 magrittr::`%>%`
 
 # Fix R CMD check notes
-#' @import shiny teal.data teal.slice teal.transform
+#' @import shiny teal.data teal.slice
```

Comment:
Is importing `teal_data` necessary after the refactor?

Reply:
Yes, packages listed in `Depends` have to be imported to the Namespace